File: test_index_babel.py
import re

with open('web/index.html', 'r') as f:
    html = f.read()

blocks = re.findall(r'<script type="text/babel">([\s\S]*?)</script>', html)
with open('index_block.jsx', 'w') as f:
    f.write('\n\n'.join(blocks))

File: test_index_babel.js
const fs = require('fs');
const html = fs.readFileSync('web/index.html', 'utf8');
const blocks = [];
const regex = /<script type="text\/babel">([\s\S]*?)<\/script>/g;
let match;
while ((match = regex.exec(html)) !== null) {
  blocks.push(match[1]);
}
fs.writeFileSync('index_block.js', blocks.join('\n\n'));
